commit 40733377ca544283d8d0444b3fadf6b08d7020ae
Author: leej3 <leej3@quansight.com>
Date:   Mon Mar 14 07:56:18 2022 -0500

    modify imports

diff --git a/examples/cwebp.c b/examples/cwebp.c
index 927063a..07b67bf 100644
--- a/examples/cwebp.c
+++ b/examples/cwebp.c
@@ -34,7 +34,7 @@
 extern "C" {
 #endif
 
-extern void* VP8GetCPUInfo;   // opaque forward declaration.
+__declspec(dllimport) extern void* VP8GetCPUInfo;   // opaque forward declaration.
 
 #ifdef __cplusplus
 }    // extern "C"
diff --git a/examples/dwebp.c b/examples/dwebp.c
index 5cfb0fb..ba84a04 100644
--- a/examples/dwebp.c
+++ b/examples/dwebp.c
@@ -33,7 +33,7 @@ static int quiet = 0;
 extern "C" {
 #endif
 
-extern void* VP8GetCPUInfo;   // opaque forward declaration.
+__declspec(dllimport) extern void* VP8GetCPUInfo;   // opaque forward declaration.
 
 #ifdef __cplusplus
 }    // extern "C"
diff --git a/src/dsp/dsp.h b/src/dsp/dsp.h
index c4f57e4..973e29d 100644
--- a/src/dsp/dsp.h
+++ b/src/dsp/dsp.h
@@ -262,6 +262,8 @@ typedef enum {
 // returns true if the CPU supports the feature.
 typedef int (*VP8CPUInfo)(CPUFeature feature);
 WEBP_EXTERN VP8CPUInfo VP8GetCPUInfo;
 
 //------------------------------------------------------------------------------
 // Init stub generator
@@ -642,7 +644,8 @@ extern void (*WebPExtractGreen)(const uint32_t* WEBP_RESTRICT argb,
 // Un-Multiply operation transforms x into x * 255 / A.
 
 // Pre-Multiply or Un-Multiply (if 'inverse' is true) argb values in a row.
-extern void (*WebPMultARGBRow)(uint32_t* const ptr, int width, int inverse);
+__declspec(dllimport) extern void (*WebPMultARGBRow)(uint32_t* const ptr, int width, int inverse);
+
 
 // Same a WebPMultARGBRow(), but for several rows.
 void WebPMultARGBRows(uint8_t* ptr, int stride, int width, int num_rows,
